jdaviz/configs/cubeviz/plugins/spectral_extraction/tests/test_spectral_extraction.py

```diff
@@ -504,3 +504,17 @@ def test_extraction_composite_subset(cubeviz_helper, spectrum1d_cube):
         (spectrum_1 + spectrum_2).flux.value,
         (spectrum_3).flux.value
     )
+
+
+def test_spectral_extraction_with_correct_sum_units(cubeviz_helper,
+                                                    spectrum1d_cube_fluxunit_jy_per_steradian):
+    cubeviz_helper.load_data(spectrum1d_cube_fluxunit_jy_per_steradian)
```

Comment:
This is not strictly necessary, but good for future quality of life. Could we add an `assert` after this line to ensure that the loaded cube's flux unit is Jy/sr? That would make it really clear what this test is checking for later on.